### PR TITLE
vnstat*: fix flag, rename placeholder and add Spanish translation

### PR DESCRIPTION
**Pull request by hasecilu**
It seems before the flag was `--top10` but now the flag is `--top` and accepts an argument.
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.es/linux/vnstat.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# vnstat
+
+> Un monitor de tráfico de red basado en consola.
+> Más información: <https://manned.org/vnstat>.
+
+- Muestra resumen de tráfico para todas las interfaces:
```
Comment (tricantivu) [inline pages.es/linux/vnstat.md:6]:
```suggestion
- Muestra un resumen de tráfico de todas las interfaces:
```

File: pages.es/linux/vnstat.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`vnstat`
+
+- Muestra resumen de tráfico para una interfaz de red específica:
```
Comment (tricantivu) [inline pages.es/linux/vnstat.md:10]:
```suggestion
- Muestra un resumen de tráfico de una interfaz de red específica:
```

File: pages.es/linux/vnstat.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`vnstat -i {{interfaz_de_red}}`
+
+- Muestra estadísticas en vivo para una interfaz de red específica:
```
Comment (tricantivu) [inline pages.es/linux/vnstat.md:14]:
```suggestion
- Muestra estadísticas en vivo de una interfaz de red específica:
```

File: pages.es/linux/vnstat.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# vnstat
+
+> Un monitor de tráfico de red basado en consola.
```
Comment (tricantivu) [inline pages.es/linux/vnstat.md:3]:
```suggestion
> Monitor de tráfico de red de línea de comandos.
```

File: pages.es/linux/vnstati.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> Compatibilidad de salida a imagen PNG para vnStat.
+> Más información: <https://manned.org/vnstati>.
+
+- Genera un resumen de los últimos 2: meses, días y todos los tiempos:
```
Comment (tricantivu) [inline pages.es/linux/vnstati.md:6]:
```suggestion
- Genera un resumen de los últimos dos meses, días, etc:
```

File: pages.es/linux/vnstati.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> Compatibilidad de salida a imagen PNG para vnStat.
```
Comment (tricantivu) [inline pages.es/linux/vnstati.md:3]:
```suggestion
> Genera imágenes PNG compatibles con vnStat.
```

Comment (hasecilu) [reply pages.es/linux/vnstati.md:3]: I think "Soporte de salida de imágenes PNG para vnStat." will be better

Comment (tricantivu) [reply pages.es/linux/vnstati.md:3]: I disagree, AFAIK, your sentence is not in an imperative mood.

File: pages.es/linux/vnstat.md
Comment (kant) [inline pages.es/linux/vnstat.md:1]: LGTM

File: pages.es/linux/vnstati.md (hunk 6 above)
Comment (kant) [reply pages.es/linux/vnstati.md:3]: I am agree with @tricantivu point of view.

Comment (hasecilu) [reply pages.es/linux/vnstati.md:3]: On "PNG image output support for vnStat."  the part "PNG image output support" acts as noun and "for vnStat" is complement, there is not really a verb to make it imperative.
Maybe "Generador de imágenes PNG compatible con vnStat.", but again is not imperative.

Comment (tricantivu) [reply pages.es/linux/vnstati.md]: I get that, however, rewriting the description as suggested might be clearer. I personally, do not like how sentences sound with `salida`  (e.g. `Genera salida`) nor other alternatives such as `Imprime datos`, `Imprime imágenes`.

**Review by tricantivu — APPROVED** [review]

**Pull request merged**